src/setup/devnet/index.js

```diff
@@ -568,15 +568,16 @@ export class Devnet {
 
                     // set heimdall peers with devnet heimdall hosts
                     for (let i = 0; i < this.totalNodes; i++) {
-                        if (this.config.devnetType === "docker" && i === 0) {
+                        if (this.config.devnetType === "docker") {
```

Comment:
Are you sure this is needed?

Reply:
For docker, we only care about 1 instance